using-timescaledb/data-retention.md

```diff
@@ -20,6 +20,46 @@ the `conditions` hypertable will still have data stretching back 36 hours.
 For more information on the `drop_chunks` function and related
 parameters, please review the [API documentation][drop_chunks].
 
+### Data Retention and Continuous Aggregates
+
+If you have a continous aggregate set up with a hypertable, it can be
+useful to keep the aggregated data but clean up old hypertable chunks
+to save memory. Since a drop of the old chunks will trigger an update
+of the continous aggregate and remove the downsampled data, it is
+necessary to set the `ignore_invalidation_older_than` parameter for
+the continous aggregate before dropping the chunks associated with the
+raw table.
+
+For example, if you have a hypertable `conditions` and have set up a
+continous aggregate `conditions_summary`:
+
+```sql
+CREATE VIEW conditions_summary WITH (timescaledb.continous) AS
+SELECT location, time_bucket('1d', "time") AS day,
+       AVG(temperature) AS avg_temperature,
+       MAX(temperature) AS max_temperature,
+       MIN(temperature) AS min_temperature
+FROM conditions
+GROUP BY location, time_bucket('1d', "time")
+```
+
+To remove chunks older than, say 30 days, you would have to first set
+the `ignore_invalidation_older_than`:
+
+```sql
+ALTER VIEW conditions_summary SET (
+  timescaledb.ignore_invalidation_older_than = '30 days'
+)
+```
+
+After that, you can drop the old chunks from the raw table
+`conditions` without triggering an update of the conditional
+aggregate:
+
+```sql
+SELECT drop_chunks(interval '30 days', 'conditions');
+```
+
 ### Automatic Data Retention Policies :enterprise_function:
```

Comment:
```suggestion
### Automatic Data Retention Policies 
```

Reply:
This might seem like a not, but I think I should handle this as a separate PR, since that require a different discussion.

Reply:
Uploaded as #326